modules/tfm/tfm/boards/include/zephyr/kernel.h

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef __ZEPHYR_KERNEL_H
+#define __ZEPHYR_KERNEL_H
+
+#include <zephyr/sys/printk.h>
+
+#define k_panic() tfm_core_panic()
```

Comment:
It will be valuable to add a comment in the header files that we override in order for the users to be able to understand the reason for this without looking at the commit message I think.

Reply:
I second this. Some documentation would be nice!

Reply:
I added a short comment in all the headers.